src/test/feature/type/TestType.cpp

```diff
@@ -0,0 +1,77 @@
+#include "gtest/gtest.h"
+
+#include "lib/sql_util.h"
+
+using std::string;
```

Comment:
It could compile without this. sql_util.h has included <string>. Do we really need to include this?